frontend/src/framework/store/middlewares/error-middleware.ts

```diff
@@ -0,0 +1,25 @@
+import { type Middleware, isRejectedWithValue } from '@reduxjs/toolkit';
+
+import {
+    NotificationStatus,
+    notify,
+} from '~/framework/services/notification-service/notification.service.js';
+
+const errorMiddleware: Middleware = () => {
+    return (next) => {
+        return (action) => {
+            if (isRejectedWithValue(action)) {
+                const { message = 'Something went wrong' } = action.error;
+
+                notify({
+                    message: message,
+                    status: NotificationStatus.Error,
+                });
```

Comment:
Application level action should be created for notifications

Reply:
A little bit didnt understand what does "application level action" means?